src/components/ToDoContainer/index.js

```diff
@@ -1,9 +1,143 @@
 import React, { Component } from "react";
 
+
+let id = 0;
+
+class Add extends Component {
+  constructor(props) {
+    super(props); 
+    this.state = {value : ''};
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({value: event.target.value});
+  }
+
+  handleSubmit(event) {
+    this.props.addTodo(this.state.value);
+    event.preventDefault();
+  }
+  
+  render () {
+    return (
+      <form onSubmit = {this.handleSubmit}>
+        <input type="text" value={this.state.value} onChange={this.handleChange}/>
+        <input type="submit" value="Add"/>
+      </form>
+    );
+  }
+}
+
+class Todoitem extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(event) {
+    this.props.deleteTodo(this.props.id);
+    event.preventDefault();
+  }
+
+  render() {
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <li>{this.props.payload}</li>
```

Comment:
Place everything here wrapped inside of the `<li>` tag since both the button and the text are part of the list element